src/components/item/ItemForm.vue

```diff
@@ -1,10 +1,12 @@
 <script>
 import InputCustom from '@/components/formElements/InputCustom.vue';
 import TextareaCustom from '@/components/formElements/TextareaCustom.vue';
-import CheckboxCustom from '@/components/formElements/CheckboxCustom.vue';
-import SelectCustom from '@/components/formElements/SelectCustom.vue';
 import ButtonText from '@/components/formElements/ButtonText.vue';
+import TogglingBlock from '@/components/wrappers/TogglingBlock.vue';
+import PopUp from '@/components/wrappers/PopUp.vue';
```

Comment:
popup is one word already ))

Reply:
it's actually written as "pop-up", so I camel-cased it this way